messaging/src/test/java/org/axonframework/commandhandling/distributed/commandfilter/AcceptAllSerializationTest.java

```diff
@@ -0,0 +1,30 @@
+package org.axonframework.commandhandling.distributed.commandfilter;
+
+import org.axonframework.serialization.TestSerializer;
+import org.junit.jupiter.params.*;
```

Comment:
Not sure what is your take on `*` imports but wanted to point it out.

Reply:
This is the style sheet doing it's thing. Fine with having a look at the sheet to be honest, might use some fine tuning.